api/app/src/command/webhook/fitbit.ts

```diff
@@ -0,0 +1,82 @@
+import { ProviderSource } from "@metriport/api";
+import { getConnectedUserByTokenOrFail } from "../connected-user/get-connected-user";
+import { getSettingsOrFail } from "../settings/getSettings";
+import { createWebhookRequest } from "./webhook-request";
+import { processRequest, WebhookUserDataPayload, reportDevicesUsage } from "./webhook";
+import dayjs from "dayjs";
+import { ConnectedUser } from "../../models/connected-user";
+import { Constants } from "../../shared/constants";
+import { capture } from "../../shared/notifications";
+import { FitbitCollectionTypes } from "../../mappings/fitbit/constants";
+
+export type FitbitWebhookNotification = {
+  collectionType: FitbitCollectionTypes;
+  date: string;
+  ownerId: string;
+  ownerType: string;
+  subscriptionId: string;
+};
+
+export const processData = async (data: FitbitWebhookNotification[]) => {
+  console.log("Starting to process the webhook");
+
+  for (const update of data) {
+    const { collectionType, date, ownerId: fitbitUserId } = update;
+
+    let userCxId;
+    try {
+      const connectedUser = await getConnectedUserByTokenOrFail(
+        ProviderSource.fitbit,
+        fitbitUserId
+      );
+
+      const cxId = connectedUser.cxId;
+      userCxId = cxId ? cxId : undefined;
```

Comment:
nit: we could just type `userCxId` as `string | undefined` and initialize it with `undefined`... then here we assign `connectedUser.cxId` to it, no need to declare `cxId` on the scope of `try` and do the ternary.